ReactiveCocoa/Swift/Signal.swift

```diff
@@ -181,14 +181,24 @@ extension SignalType {
 		return observe(Observer(action))
 	}
 
-	/// Observes the Signal by invoking the given callback when `next` events are
-	/// received.
+	@available(*, unavailable, message="This Signal may emit errors which must be handled explicitly, or observed using observeResult:")
```

Comment:
I think unavailable is correct, this is actually sort of a bug fix. If this breaks a client... Then good! We helped someone find a potential bug :) it's not like we're not allowing them from using the method anymore, they just need to start observing errors or handling them.

Does that make sense?